cypress/support/fragments/data_import/logs/logsViewAll.js

```diff
@@ -120,13 +122,11 @@ export default {
   },
 
   selectYesfilterJobsByErrors: () => {
-    cy.do(Accordion('Errors in import')
-      .find(Checkbox({ id: 'clickable-filter-statusAny-error' })).click());
+    cy.do(errorsInImportAccordion.find(Checkbox({ id: 'clickable-filter-statusAny-error' })).click());
```

Comment:
Checkbox also to const

Reply:
All Checkboxes are different. What Checkbox do you mean?